### PR TITLE
feat: Configurable repos.csv path

### PR DESCRIPTION
**Pull request by pstreef**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ The org structure is represented in a series of columns `org1..orgN` where `N` i
 
 There is repetition in organizational hierarchy in this format. The reference implementation validates the internal consistency of those organizational repositories across CSV lines.
 
+You can either overwrite the existing [repos.csv](src/main/resources/repos.csv) or set the `moderne.scm.reposCsvPath` in [application.yaml](src/main/resources/application.yaml).
```
Comment (kmccarp) [inline README.md:57]: Does it make sense for it to be in the `scm` namespace?

Comment (pstreef) [reply README.md:57]: hmm, no actually not 🤔

Comment (pstreef) [reply README.md:57]: I will extract it

File: README.md [hunk 2]
```diff
@@ -54,6 +54,8 @@ The org structure is represented in a series of columns `org1..orgN` where `N` i
 
 There is repetition in organizational hierarchy in this format. The reference implementation validates the internal consistency of those organizational repositories across CSV lines.
 
+You can either overwrite the existing [repos.csv](src/main/resources/repos.csv) or set the `moderne.reposCsvPath` in [application.yaml](src/main/resources/application.yaml).
```
Comment (kmccarp) [inline README.md:57]:
```suggestion
You can either overwrite the existing [repos.csv](src/main/resources/repos.csv), set the `moderne.reposCsvPath` in [application.yaml](src/main/resources/application.yaml), or provide the path via an argument on application startup.
```

**Review by kmccarp — APPROVED** [review]

**Pull request merged**